app/src/main/res/layout-sw600dp/revision_card_fragment.xml

```diff
@@ -20,162 +20,156 @@
+      android:orientation="vertical"
+      android:paddingBottom="@dimen/revision_card_fragment_padding_bottom">
+
+      <TextView
+        android:id="@+id/revision_card_explanation_text"
+        style="@style/Body"
+        android:layout_marginStart="@dimen/revision_card_fragment_layout_margin_start"
+        android:layout_marginTop="@dimen/revision_card_fragment_layout_text_margin_top"
+        android:layout_marginEnd="@dimen/revision_card_fragment_layout_margin_end"
+        android:textColorLink="@color/component_color_shared_link_text_color" />
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/revision_card_fragment_navigation_card_container"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:minWidth="@dimen/revision_card_fragment_layout_tablet_min_width"
-        android:orientation="vertical"
-        android:paddingBottom="@dimen/revision_card_fragment_padding_bottom">
+        android:layout_marginTop="50dp">
 
         <TextView
-          android:id="@+id/revision_card_explanation_text"
-          style="@style/Body"
-          android:layout_marginStart="@dimen/revision_card_fragment_layout_margin_start"
-          android:layout_marginTop="@dimen/revision_card_fragment_layout_text_margin_top"
-          android:layout_marginEnd="@dimen/revision_card_fragment_layout_margin_end"
-          android:textColorLink="@color/component_color_shared_link_text_color" />
-
-        <androidx.constraintlayout.widget.ConstraintLayout
-          android:id="@+id/revision_card_fragment_navigation_card_container"
+          android:id="@+id/continue_studying_text_view"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginTop="50dp">
-
-          <TextView
-            android:id="@+id/continue_studying_text_view"
+          android:layout_marginStart="28dp"
+          android:layout_marginEnd="28dp"
+          android:text="@string/revision_navigation_cards_header"
+          android:textAlignment="@{viewModel.subtopicId == 1 ? View.TEXT_ALIGNMENT_TEXT_END : View.TEXT_ALIGNMENT_TEXT_START}"
+          android:textColor="@color/component_color_shared_primary_text_color"
+          android:textSize="20sp"
+          android:textStyle="bold"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="parent" />
+
+        <com.google.android.material.card.MaterialCardView
+          android:id="@+id/previous_navigation_card"
+          android:layout_width="144dp"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="28dp"
+          android:layout_marginTop="16dp"
+          android:layout_marginEnd="28dp"
+          android:layout_marginBottom="@dimen/topic_revision_summary_view_container_margin_bottom"
+          android:clipToPadding="true"
```

Comment:
I think clipToPadding="true" is default for MaterialCardView and can be ommitted.